steward/src/commands.rs

```diff
@@ -85,13 +85,22 @@ impl Configurable<StewardConfig> for EntryPoint {
             .map(PathBuf::from)
             .unwrap_or_else(|| CONFIG_FILE.into());
 
-        if filename.exists() {
+        let config_env_variable = option_env!("STW_ENV_VAR");
+
+        if config_env_variable != None {
+            let new_filename = self.config.as_ref().map(PathBuf::from).unwrap_or_else(|| {
+                config_env_variable
+                    .expect("Can't process config_env_variable")
+                    .into()
+            });
+            println!("{}", new_filename.display());
```

Comment:
I assume this was here for debugging purposes but we shouldn't print out the filename here.

Reply:
done